generate.py

```diff
@@ -130,38 +146,40 @@ def make_payment_op(account_id, amount):
 def main(inflation):
 	# TODO: Let user select the connection type
 	# The stellar/quickstart Docker image uses PostgreSQL
-	conn = sqlite3.connect(db_address)
+	#conn = sqlite3.connect(db_address)
+	conn = psycopg2.connect(db_address)
 
 	# Get the next sequence number for the transactions
 	sequence = horizon.account(pool_address).get('sequence')
 	inflation = XLM_Decimal(inflation)
-	transactions = []
 	total_payments_cost = 0
 	num_payments = 0
 	total_fee_cost = 0
 
 	# Create one transaction for each batch
+	transactions = []
 	for batch in accounts_payouts(conn, pool_address, inflation):
 		op_count = 0
 		ops = {'sequence': sequence, 'operations': []}
 		for aid, amount in batch:
-			# Check if the payment destination (aid) is valid
-			try:
-				acc = horizon.get(aid)
-			except AccountNotExistError:
-				continue
-			if not acc:
-				continue
+			# Check if the payment destination (aid) is valid (TOO SLOW!)
+			#try:
+			#	acc = horizon.account(aid)
+			#except AccountNotExistError:
+			#	continue
+			#if not acc:
+			#	continue
 			# Include payment operation on ops{}
-			ops['operations'].append(aid, amount)
+			ops['operations'].append(make_payment_op(aid, amount))
 			op_count += 1
 
 		# Build transaction
 		tx = Transaction(
-			source=pool_address,
-			opts=ops
+			source = pool_address,
```

Comment:
We should not use whitespace around operator  in parameters

https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements